src/main/kotlin/com/zepben/evolve/services/network/NetworkService.kt

```diff
@@ -58,6 +62,9 @@ class NetworkService : BaseService("network") {
     fun add(cableInfo: CableInfo): Boolean = super.add(cableInfo)
     fun remove(cableInfo: CableInfo): Boolean = super.remove(cableInfo)
 
+    fun add(currentTransformerInfo: CurrentTransformerInfo): Boolean = super.add(currentTransformerInfo)
+    fun remove(currentTransformerInfo: CurrentTransformerInfo): Boolean = super.remove(currentTransformerInfo)
+
```

Comment:
todo: You have merged these into the wrong package subsection. Should be under `IEC61968 infIEC61968 InfAssetInfo`